.github/workflows/rbc_test.yml

```diff
@@ -256,8 +190,20 @@ jobs:
           mamba run -n docker docker-compose logs --no-color --tail=10000 -f -t \> omniscidb-docker.log
           cat omniscidb-docker.log
 
-      - name: Show Omniscidb server output [conda]
+      - name: Stop Omniscidb server [docker]
         shell: bash -l {0}
-        if: matrix.os == 'ubuntu-latest' && matrix.omniscidb-from == 'conda'
+        timeout-minutes: 1
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.omniscidb-from == 'docker' }}
         run: |
-          cat omniscidb-output.txt
+          mamba run -n docker docker-compose stop
+          sleep 10
+
+      - name: Run RemoteJIT tests
+        shell: bash -l {0}
+        env:
+          EXPECTED_PYTHON_VERSION: ${{ matrix.python-version }}
+          EXPECTED_OMNISCIDB_VERSION: ${{ matrix.omniscidb-version }}
```

Comment:
nit: Running omnisci related tests are disabled here, hence there is no need to define `EXPECTED_OMNISCIDB_VERSION`